src/main/kotlin/nl/info/zac/smartdocuments/rest/RestSmartDocumentsTemplateGroup.kt

```diff
@@ -49,3 +49,14 @@ private fun convertTemplateGroupResponseToRest(
         groups = group.templateGroups?.map { convertTemplateGroupResponseToRest(it) }?.toSet(),
         templates = group.templates?.map { RestSmartDocumentsTemplate(it.id, it.name) }?.toSet()
     )
+
+fun Set<RestSmartDocumentsTemplateGroup>?.group(groupName: String) =
+    this?.find { it.name == groupName }
+
+fun Set<RestSmartDocumentsTemplateGroup>?.group(groupNames: List<String>): RestSmartDocumentsTemplateGroup? {
```

Comment:
Same here. I don't see the need for the null check here. Makes the code quite unclear to me.